src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -388,16 +388,16 @@ def test_user_standard_waveform(session):
     session.clear_user_standard_waveform()
 
 
-# TODO(bhaswath): Doesn't work, issue #596
-'''
 def test_fir_filter_coefficients():
     with nifgen.Session('', '0', False, 'Simulate=1, DriverSetup=Model:5441;BoardType:PXI') as session:
-        coeff_array = [1, 0, -1]
+        coeff_array = [0 for i in range(95)]
+        coeff_array[0] = -1.0
+        coeff_array[2] = 1.0
```

Comment:
This should error because the device expected symmetrical coefficients.
I think you need to also enable OSP so that the OSP configuration gets validated.

Reply:
Test disabled since OSP disabled, can't check this.